tests/test_cli_gui/test_search.py

```diff
@@ -139,12 +139,18 @@ def test_real_search():
     )
     assert response_list.status_code == 200
     data = json.loads(response_list.get_data(as_text=True))
-    assert len(data) == 6
+    assert len(data) == 7
     i = 0
 
     assert data[i]["id"] == "fetchai/gym:0.1.0"
     assert data[i]["description"] == "The gym connection wraps an OpenAI gym."
     i += 1
+    assert data[i]["id"] == "fetchai/http:0.1.0"
+    assert (
+        data[i]["description"]
+        == "The HTTP connection that wraps a web-based, RESTful API specification."
+    )
+    i += 1
```

Comment:
You have duplicated the same block making your tests fail. Removing this one and keeping the one below should make the tests pass.

Reply:
that is fixed, thanks